code/modules/surgery/organs/_organ.dm

```diff
@@ -66,6 +66,8 @@
 	var/list/organ_effects
 	/// String displayed when the organ has decayed.
 	var/failing_desc = "has decayed for too long, and has turned a sickly color. It probably won't work without repairs."
+	/// Assoc list of alternate zones where this can organ be slotted to organ slot for that zone
+	var/list/valid_zones = null
```

Comment:
lazylist?

Reply:
Not really? Its define-only, there are no cases where we need to treat it like one in-code.